examples/with-react-native-web/packages/ui/package.json

```diff
@@ -16,7 +16,7 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
-    "react": "18.1",
+    "react": "^18.1.0",
```

Comment:
Latest published version has a strict peer dependency on 18.1: https://github.com/facebook/react-native/blob/v0.70.4/package.json#L96-L98, but it's updated in main: https://github.com/facebook/react-native/blob/07bd590843bf84c0196086f56f46ebd0d236b1ef/package.json#L107-L109. Either way, the run-example test is passing with these updates:

```
pnpm turbo run-example -- with-react-native-web npm
```